x-pack/test/accessibility/apps/painless_lab.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'security']);
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const a11y = getService('a11y');
+
+  describe('Accessibility Painless Lab Editor', () => {
+    before(async () => {
+      await PageObjects.common.navigateToApp('painlessLab');
+      await a11y.testAppSnapshot();
+      expect(await testSubjects.exists('painless_lab')).to.be(true);
```

Comment:
I don't think this expect and await needs to be here because we have already navigated to painless lab?  I would keep this line in before sync 
```
await PageObjects.common.navigateToApp('painlessLab'); 
```
and then in the first test I will just need to make sure the already open painless lab meets a11y specifications.

Reply:
yes, makes sense , removed it .